src/components/Fingerprint.vue

```diff
@@ -0,0 +1,113 @@
+<template>
+	<div 
+		class="wikibase-termbox-fingerprint" 
+		:class="modifier()">
```

Comment:
Vue can do such mapping via
```suggestion
		:class="{ 'wikibase-termbox-fingerprint--primaryLanguage': isModified }">
```

However I did not quite get behind the naming of `isModified` - what's the story?